src/components/home-english.jsx

```diff
@@ -0,0 +1,171 @@
+"use strict";
+
+import $ from "jquery";
+import _ from "lodash";
+import React from "react";
+import {Link} from "react-router";
+import {Grid, Row, Col, Glyphicon} from "react-bootstrap";
+
+import Header from "./header";
+import NewsContainer from "./news/news-container";
+
+import "./home.less";
+import Google from "../../assets/img/sponsors/google.png";
+import Intel from "../../assets/img/sponsors/intel.png";
+import Orange from "../../assets/img/sponsors/orange.png";
+import EasyHost from "../../assets/img/sponsors/easyhost.png";
+import Cisco from "../../assets/img/sponsors/cisco.png";
+import Intuitext from "../../assets/img/sponsors/intuitext.gif";
+import Apdetic from "../../assets/img/sponsors/apdetic.png";
+import TotalSoft from "../../assets/img/sponsors/totalsoft.gif";
+import Upir from "../../assets/img/sponsors/upir.png";
+import MEN from "../../assets/img/sponsors/edu.jpg";
+
+
+export default React.createClass({
+  displayName: "Home",
+
+  componentDidMount() {
+    this.props.changeLanguage("en");
+  },
+
+  render() {
+    return <div className="home">
```

Comment:
Pune aici `className="home english"`.

Apoi in `home.less`. Aliniaza butonul ala cu `About`.